lib/builders/documentBlueprints/documentBlueprintsValueBuilder.ts

```diff
@@ -1,17 +1,23 @@
-﻿import { DocumentBlueprintsBuilder } from "./documentBlueprintsBuilder";
+﻿import {BlockGridValueBuilder} from "../document/blockGridValueBuilder";
+import {BlockListValueBuilder} from "../document/blockListValueBuilder";
+import {DocumentBlueprintsBuilder} from "./documentBlueprintsBuilder";
 
 export class DocumentBlueprintsValueBuilder {
   parentBuilder: DocumentBlueprintsBuilder;
-  culture: string;
+  culture: string | null;
   segment: string;
   alias: string;
-  value: string;
+  value: string | string[];
+  editorAlias: string;
+  entityType: string;
+  blockGridValueBuilder: BlockGridValueBuilder;
+  blockListValueBuilder: BlockListValueBuilder;
 
   constructor(parentBuilder: DocumentBlueprintsBuilder) {
     this.parentBuilder = parentBuilder;
   }
 
-  withCulture(culture: string) {
+  withCulture(culture: string | null) {
```

Comment:
When we don't define a culture, it is set to null. Is there a reason to also set it to null in the withCulture?